.gitignore

```diff
@@ -26,6 +26,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+_scm_version.py
```

Comment:
We should only ignore this module in a specific place of the repository, on in any location.
Please use `/src/sphinxcontrib/towncrier/_scm_version.py` instead.

Also, it is better to put it at the end of this file because it is generated from some templates and if we'll want to re-generate it, it'll get lost. Plus, in general, it's normally better to keep project-specific entries in a separate section.

Reply:
Got it. Fix pushed.